snuba/snuba_migrations/uptime_monitor_checks/0001_uptime_monitor_checks.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set = StorageSetKey.UPTIME_MONITOR_CHECKS
```

Comment:
I think you need to use the EAP storage set.

You can keep the migration group as uptime monitor checks.

Reply:
````
The storage_set=StorageSetKey.EVENTS_ANALYTICS_PLATFORM is present in more than one MigrationGroup. Each storage_set should only reference one migration group
````

doesn't seem to like that

Reply:
Eh, I was wrong, you can't keep it.

You should move the migration to the events_analytics_platform directory. The benefit is your migration will be run automatically on the right clusters since the dataset is configured everywhere.

Reply:
👍 will do so shortly